portal-frontend/src/app/features/notice-of-intents/edit-submission/additional-information/additional-information.component.ts

```diff
@@ -170,7 +175,7 @@ export class AdditionalInformationComponent extends FilesStepComponent implement
           newForm.addControl(`${lot.id}-area`, new FormControl(lot.area, [Validators.required]));
         }
         this.structuresForm = newForm;
-
+        
```

Comment:
I would recommend installing [Prettier](https://prettier.io/) to automatically detect and remove tabs.